comp/trace/config/setup_serverless.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build serverless
+
+package config
+
+func containerTagsFunc(cid string) ([]string, error) {
```

Comment:
I think the linter might complain about the unused argument
```suggestion
func containerTagsFunc(_ string) ([]string, error) {
```